test/spec/modules/pubmaticBidAdapter_spec.js

```diff
@@ -448,6 +449,197 @@ describe('PubMatic adapter', function () {
   		  expect(data.imp[0].ext.pmZoneId).to.equal(bidRequests[0].params.pmzoneid.split(',').slice(0, 50).map(id => id.trim()).join()); // pmzoneid
   		});
 
+      it('Request should have digitrust params', () => {
```

Comment:
Can you update the arrow functions in the various test functions to use the `, function() {` syntax?  This is more in-line with Mocha's style/expectations.

Reply:
I have made these changes now.